xradar/io/export/cfradial1.py

```diff
@@ -304,6 +306,21 @@ def to_cfradial1(dtree=None, filename=None, calibs=True):
 
     dataset.attrs = dtree.attrs
 
+    # Find variables with >= 2 dimensions
+    multidim_variables = [
```

Comment:
Do we need to check if we have `azimuth`/`range`, `elevation`/`range` or `time`/`range` as dimensions?

Reply:
I don't think we need to confirm these dims explicitly, although it will be better if we do that, but again we are doing that in the _variable_mapper function. Generally in the CfRadial1 data, there will be data variables with the same shape based on the number of rays (nrays) and gates (ngates). It doesn't matter whether these are base moments, polarimetric, or even reterivals, they should have this fill value. But, if you would like to see it that way, then i can add that condition easily.

Reply:
Great, let's keep this as is and react if needed.